frontend/src/components/VDetailOrganisation.vue

```diff
@@ -21,78 +19,78 @@
 
     <div class="card mt-2">
       <div class="card-body">
-        <b-list-group-item variant="dark">
+        <b-list-group-item class="bg-light">
           {{ $t('department') }}
         </b-list-group-item>
-        <b-list-group-item>
+        <b-list-group-item class="mt-2">
           <v-tree-view/>
         </b-list-group-item>
       </div>
     </div>
 
     <div v-if="result.managers.length" class="card mt-2">
```

Comment:
Notice how you repeat yourself for both managers, employees and associated. The same pattern is reapeated for employees. I think it makes sense to make this cunk into a component so you can make easier changes to the view.
To see how you can still add custom data into the component, checkout slots https://vuejs.org/v2/guide/components-slots.html